website/content/docs/nia/api/health.mdx

```diff
@@ -0,0 +1,48 @@
+---
+layout: docs
+page_title: Consul-Terraform-Sync Health API
+description: >-
+  Consul-Terraform-Sync Health API
+---
+
+# Health
+
+The `/health` endpoints provide information on the health of a running Consul-Terraform-Sync (CTS) instance.
```

Comment:
Should I replace `instance` with `service` ?

Reply:
I guess I lean slightly more towards "instance" but only because when I search our docs for "CTS instance" there are two results. Searching for "CTS service" didn't return any precedents. Open to what others think

Reply:
I also lean towards instance because I think we'll eventually start referring to them as instances when HA is introduced. 

This is a side note too, but maybe this should be "endpoint" (singular).